hack/externalnode/install-vm.sh

```diff
@@ -120,10 +134,30 @@ Restart=on-failure
 WantedBy=multi-user.target
 EOF
     systemctl daemon-reload
-    systemctl enable antrea-agent
-    echo "Starting antrea-agent service"
-    systemctl start antrea-agent
-    systemctl status antrea-agent
+    systemctl enable "$ANTREA_AGENT"
+    echo "Starting ${ANTREA_AGENT} service"
+    systemctl start "$ANTREA_AGENT"
+    systemctl status "$ANTREA_AGENT"
+}
+
+check_ovs_config_and_cleanup() {
+    bridges=$(ovs-vsctl list-br)
+    for br in $bridges; do
+        if [ "$br" != "$OVS_BRIDGE" ] ; then
+            continue
+        fi
+        # Check if any of the interface is in error state.
+        ports=$(ovs-vsctl list-ports $OVS_BRIDGE)
```

Comment:
Why the outer for-loop is needed? Why not start from this line? The same for Windows implementation.

Reply:
I want to make sure that ovs bridge that we are looking actually exists. If it does not exist, then we don't need to do anything.